src/linter/block_linter.go

```diff
@@ -297,14 +297,14 @@ func (b *blockLinter) checkBinaryDupArgs(n, left, right ir.Node) {
 // checkVoidType reports if node has void type
 func (b *blockLinter) checkVoidType(n ir.Node) {
 	if b.walker.exprType(n).Is("void") {
-		b.report(n, LevelDoNotReject, "voidResultUsed", "void function result used")
+		b.report(n, LevelNotice, "voidResultUsed", "void function result used")
```

Comment:
This is usually a serious logical error and it's easily fixed with a discard var assignment.
I would mark it as error as it will cause a program to misbehave.